docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -205,10 +205,13 @@ Image=quay.io/centos/centos
 
 ### Relative paths
 
-In order to support Systemd specifiers, Quadlet does not resolve relative paths that start with `%`.
-To resolve such a path, prepend it with `./`.
+In order to support Systemd specifiers, Quadlet does not resolve relative paths
+that start with `%`.  To resolve such a path, prepend it with `./`.
 
-For example, instead of `EnvironmentFile=%n/env` use `EnvironmentFile=./%n/env`
+For example, instead of `EnvironmentFile=%n/env` use
+`EnvironmentFile=./%n/env`. Keep in mind that unit files are generated in
+`$XDG_RUNTIME_DIR/systemd/generator` and environmental files will not be copied
+there.
```

Comment:
It is not clear what you want to say with this sentence. Even if they will be copied a relative path would not work because as root systemd uses `/` as working directory and as user it used the home directory by default.

In order to allow relative path in options quadlet will even resolve the path for you to the source directory so the point about copy seems pointless? It only doesn't do that for systemd specifiers